wave_matching/src/ndt.cpp

```diff
@@ -0,0 +1,63 @@
+#include "wave/utils/config.hpp"
+#include "wave/utils/logging.hpp"
+#include "wave/matching/ndt.hpp"
+
+namespace wave {
+
+NDTMatcher::NDTMatcher(float res, const std::string &config_path) {
+    this->ref = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+    this->target = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+    this->final = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
```

Comment:
Are you using the clang-format? I don't remember the settings allowing spaces at the last angle bracket ">"